pkg/distributor/distributor_test.go

```diff
@@ -1426,63 +1426,78 @@ func TestDistributor_Push_ExemplarValidation(t *testing.T) {
 	}
 
 	tests := map[string]struct {
-		req    *mimirpb.WriteRequest
-		errMsg string
-		errID  globalerror.ID
+		req       *mimirpb.WriteRequest
+		expDrop   bool
```

Comment:
Can we use `expectedX` here instead of `exp`? I thought this might be `exemplarDrop` at first

Reply:
Done.